src/components/PoolInfoCard.tsx

```diff
@@ -61,73 +69,96 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
   }
 
   return (
-    <div className="poolInfoCard">
+    <Box>
       {underlyingPool ? (
         <Tooltip title={<React.Fragment>{t("metapool")}</React.Fragment>}>
-          <h4 className="underline">{formattedData.name}</h4>
+          <Typography
+            variant="h1"
+            sx={{
+              borderBottom: "1px dotted",
+              width: "fit-content",
+              cursor: "help",
+            }}
+          >
+            {formattedData.name}
+          </Typography>
         </Tooltip>
       ) : (
-        <h4>{formattedData.name}</h4>
+        <Typography variant="h1">{formattedData.name}</Typography>
       )}
-      <div className="info">
-        <div className="infoItem">
-          <span className="label bold">{`${t("status")}:`}</span>
-          <span className="value">{`${
-            data?.isPaused ? t("paused") : t("active")
-          }`}</span>
-        </div>
-        <div className="infoItem">
-          <span className="label bold">{`${t("fee")}:`}</span>
-          <span className="value">{formattedData.swapFee}</span>
-        </div>
-        <div className="infoItem">
+      <InfoItem mt={3}>
+        <Typography>{`${t("status")}:`}</Typography>
+        <Typography variant="subtitle1">{`${
+          data?.isPaused ? t("paused") : t("active")
+        }`}</Typography>
+      </InfoItem>
+      <InfoItem>
+        <Typography>{`${t("totalLocked")}:`}</Typography>
+        <Typography variant="subtitle1">{`$${formattedData.reserve}`}</Typography>
```

Comment:
```suggestion
        <Typography>{`${t("totalLocked")}: `} <Typography variant="subtitle1">{`${formattedData.reserve}`}</Typography></Typography>
```

Can you see if the formatting works fine with this structure? Also this had an extra `$`